exchanges/okx/okx_websocket.go

```diff
@@ -51,15 +51,20 @@ var (
 const (
 	// wsOrderbookChecksumDelimiter to be used in validating checksum
 	wsOrderbookChecksumDelimiter = ":"
```

Comment:
rm? becuase you addded 
<img width="289" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/99053a44-e333-42e0-8b29-99548470d6e9">